src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -19,13 +20,28 @@ class BookingFacade(
     private val log = LoggerFactory.getLogger(this::class.java)
   }
 
-  fun create(booking: CreateVideoBookingRequest, context: BvlsRequestContext): Long =
-    createVideoBookingService.create(booking, context.username).let { newBooking ->
-      sendNewCourtBookingEmail(newBooking)
-      newBooking.videoBookingId
+  fun create(bookingRequest: CreateVideoBookingRequest, username: String): Long {
+    val (booking, prisoner) = createVideoBookingService.create(bookingRequest, username)
+
+    when (bookingRequest.bookingType!!) {
+      BookingType.COURT -> sendNewCourtBookingEmail(booking, prisoner)
+      BookingType.PROBATION -> sendNewProbationBookingEmail(booking, prisoner)
     }
 
-  private fun sendNewCourtBookingEmail(newBooking: VideoBooking) {
+    return booking.videoBookingId
+  }
+
+  private fun sendNewCourtBookingEmail(booking: VideoBooking, prisoner: Prisoner) {
     log.info("TODO - send new court booking email.")
+
+    // Agreed with Tim will be using a ContactService to pull back the necessary contact information related to the booking just created.
+    // Multiple contacts will result in multiple emails i.e. one email per contact.
```

Comment:
Getting the owner email will be a later addition at present, so I'm getting the court, prison, probation and 3rd party contact details initially. I'll add it the owner email afterwards. `BookingContactsService.getBookingContacts(videoBookingId)` - just on the integration test, so coming very shortly.